lib/fog/compute/voxel.rb

```diff
@@ -1,23 +1,21 @@
-require 'fog/voxel'
-
 module Fog
   module Compute
     class Voxel < Fog::Service
-      autoload :Image, 'fog/compute/voxel/image'
-      autoload :Images, 'fog/compute/voxel/images'
-      autoload :Server, 'fog/compute/voxel/server'
-      autoload :Servers, 'fog/compute/voxel/servers'
+      autoload :Image, File.expand_path('../voxel/models/image', __FILE__)
+      autoload :Images, File.expand_path('../voxel/models/images', __FILE__)
+      autoload :Server, File.expand_path('../voxel/models/server', __FILE__)
+      autoload :Servers, File.expand_path('../voxel/models/servers', __FILE__)
 
       requires :voxel_api_key, :voxel_api_secret
       recognizes :host, :port, :scheme, :persistent
 
-      model_path 'fog/compute/voxel'
+      model_path 'fog/compute/voxel/models'
       model       :image
       collection  :images
       model       :server
       collection  :servers
 
-      request_path 'fog/compute/voxel/real'
+      request_path 'fog/compute/voxel/requests'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.